src/components/Menu-Badge/MenuBadge.constraints.ts

```diff
@@ -0,0 +1,4 @@
+export const COLOR = {
+  primary: 'linear-gradient(60deg, #B33DEB 13.4%, #DE8FFF 86.6%)',
```

Comment:
Hardcoded CSS linear-gradient string is fine for CSS use, but this gradient format can't be used inside an SVG <linearGradient/> element. You currently duplicate gradient information in the SVG markup; centralize gradient definition or derive both from a single source.

**Sugerencia:**
```typescript
If you need both an SVG gradient and a CSS gradient, store stop colors/constants here and build the SVG linearGradient dynamically or create a helper that converts the stop colors to both CSS gradient and SVG gradient. Example:

export const GRADIENT_STOPS = ['#B33DEB', '#DE8FFF'];

Then use it for CSS and generate <stop/> elements from it for SVG.
```